modules/ROOT/pages/customize.adoc

```diff
@@ -1,101 +1,177 @@
 the Salesforce Community Cloud https://help.salesforce.com/articleView?id=rss_component_reference_table.htm&type=5[Community Components, role=external, window=blank] documentation.
 
-Components that are typically used in *Home* pages include:
+Components typically used in *Home* pages include:
 
 * https://help.salesforce.com/articleView?id=rss_headline.htm&type=5[Headline, role=external, window=blank]
-* https://help.salesforce.com/articleView?id=rss_setup_nav_menu.htm&type=5[Navigation Menu, role=external, window=blank]: You can update *Max Items to Display (2-15)*,
-*Menu Item Alignment*, and use the *Navigation Menu* to display, add, remove, and customize options. *Home*
-is always present and is always the first item. The following properties can be customized:
-** *Name*
-** *Type*
-** *Page*
-** *Publicly available*
 * https://help.salesforce.com/articleView?id=rss_cms_collection.htm&type=5[CMS Collection, role=external, window=blank]
-* *API Carousel*: Displays APIs and API versions that are available to the community from Anypoint Exchange.
-* *API Details Button*: Provides details about an API.
+* *API Carousel*
++
+Displays APIs and API versions that are available to the community from Anypoint Exchange.
++
+* *API Details Button*
++
+Provides details about an API.
++
 * https://help.salesforce.com/articleView?id=community_builder_rich_text_component.htm&type=5[Rich text editor, role=external, window=blank]
 * https://help.salesforce.com/articleView?id=rss_html_editor.htm&type=5[HTML editor, role=external, window=blank]
-
-Components that are typically used in *API details* pages include:
-
-* *API Header*: Specifies the API name and description.
-* *Tabs*: Contains the following components: *API Documentation Viewer*, which displays API Exchange information,
-*API Console*, which renders information from the API RAML specification, and *Feed*, which displays comments,
-discussions, and questions.
-* *API Access Requestor*: Registers a client application and requests access to the API.
-* *Learn More Button*: Provides additional information about the API.
-
-Components that are typically used in *My Application* pages include:
-
-* *Application Listing*: Lists all client applications that the member has registered and can access.
-
-To reorder items on a page, simply drag and drop them.
+* https://help.salesforce.com/articleView?id=rss_setup_nav_menu.htm&type=5[Navigation Menu, role=external, window=blank]
+
+The *Navigation Menu* component provides the menu for navigating the site, with
+settings for *Max Items to Display (2-15)* and *Menu Item Alignment*, and a
+*Navigation Menu* button to edit the links on the navigation menu. The *Home*
+link is always present and is always the first item. Other links can be added,
+removed, and reordered. Each link other than *Home* can be customized with these
```

Comment:
passive
GLOBALLY review for passive voice, and determine whether you intend to use it or not. See style ref.